testrunner-running/src/main/scala/edu/illinois/cs/testrunner/execution/Utils.java

```diff
@@ -4,27 +4,46 @@
 import java.util.regex.Pattern;
 
 public class Utils {
-    private static final String junit5Regex = "\\[class:([\\w.]+).*\\[method:(\\w+)\\(";
-    private static final Pattern junit5Pattern = Pattern.compile(junit5Regex);
+    private static final String junit5NestedRegex = "\\[nested-class:([\\w.]+)\\]";
+    private static final Pattern junit5NestedPattern = Pattern.compile(junit5NestedRegex);
+    private static final String junit5BasicRegex = "\\[class:([\\w.]+).*\\[method:([\\w().]+)";
+    private static final Pattern junit5BasicPattern = Pattern.compile(junit5BasicRegex);
     private static final String junit4Regex = "\\[test:(\\w+)\\(([\\w.]+)\\)";
     private static final Pattern junit4Pattern = Pattern.compile(junit4Regex);
 
     /**
      * Turn the uniqueId from identifier into fully qualified method name.
      *
+     * For JUnit 5 nested test:
```

Comment:
Similarly, please put the example for the nested regex below the basic one, as the basic one is more common.

Reply:
Done